.github/workflows/ci.yaml

```diff
@@ -0,0 +1,47 @@
+name: ci
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the main branch
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    # The type of runner that the job will run on
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          # Commented as known to fail passing the tests
+          # - windows-latest
```

Comment:
I'm pretty sure they should be visible as a reference and fixed.

Reply:
I would not add until we get the checks green and fixing would be outside the scope of this change. If we include them and
ignore checks all PRs will report red, making much harder
to identify those that are ready for review or need more work.

That is due to how checks work, any red one makes the entire check red, even if is not required for merging. Its purely due
to the visual hint.

I added bug for tracking that #26